schemas/software.yml

```diff
@@ -0,0 +1,25 @@
+# Schema for all yml files in our-work/software and our-work/workshops-and-talks 
+title: str()
+description: str()
+groups: list(include('group'))
+languages: list(include('language'), required=False)
+tags: list(include('tag'))
+people: list(include('person'))
+links: list(include('link'), required=False)
+---
+language: regex('^(python|julia|R|matlab|javascript|typescript|vuejs|reactjs|rust)$')
+tag: regex('^(behavioral-tasks|neuroimaging|neuroscience)$')
+#people involved in the project
+person:
+  name: str()
+  github_user: str(required=False)
+# This string should match the identifying the slug in the collaborations section
+group: str()
+# These are links associated with the software 
+link:
+  # Must match one of the values in the regex
+  category: regex('^(code|docs|website)$')
+  url: str()
```

Comment:
in some places we use `url` and some we use `href` - should go with one or the other